Common/src/grid_adaptation_structure.cpp

```diff
@@ -42,211 +42,211 @@ CGridAdaptation::CGridAdaptation(CGeometry *geometry, CConfig *config) {
   size = SU2_MPI::GetSize();
   rank = SU2_MPI::GetRank();
 
-	unsigned long iPoint;
```

Comment:
40% of the changes are in this file, I had to replace tabs because of Codefactor